components/definitions/rewardsTip.d.ts

```diff
@@ -64,4 +62,10 @@ declare namespace RewardsTip {
     publisherKey?: string
     monthlyDate?: number
   }
+
+  export interface Balance {
```

Comment:
Do we have any way to avoid defining this type in three different places? Seems like we could benefit from a file for shared types, but not sure if that's doable.

Reply:
good point, we could do this in a separate issue